ecosystem/sep-0001.md

```diff
@@ -38,6 +38,8 @@ It is also recommended to set a `text/plain` content type so that browsers rende
 content-type: text/plain
 ```
 
+`stellar.toml` can have a maximum file size of 100kB.
```

Comment:
```suggestion
`stellar.toml` can have a maximum file size of 100KB.
```

I think uppercase `K` is more commonly used in computer data measurements.

According to [Wikipedia](https://en.wikipedia.org/wiki/Kilobyte) we should write this as `100KB` rather than `100kB` because lowercase `k` is from SI units and is `1000` and uppercase `K` is more common used for binary units and is from IEC binary units, `1024`. See 

The value in the code is `100 * 1024`:

https://github.com/stellar/go/blob/fd815d6/clients/stellartoml/main.go#L5-L6

Reply:
thanks i was wondering about the kCase